### ISSUE TITLE
[BUG] PCM0P rate_min error in sof-cml-rt1011-rt5682.conf

### ISSUE THREAD

**Issue by aiChaoSONG** [thesofproject/sof#2377]
seems we have some error in macro expansion.
```
#
# PCM Configuration
SectionPCMCapabilities."Passthrough Playback 0" {

	formats "S32_LE,S24_LE,S16_LE"
	rate_min "SSP"
	rate_max "48000"
	channels_min "2"
	channels_max "2"
	periods_min "2"
	periods_max "16"
	period_size_min	"192"
	period_size_max	"16384"
	buffer_size_min	"65536"
	buffer_size_max	"65536"
}
```

**Comment by lgirdwood** [comment]
@aiChaoSONG can you provide details, which topology, can you fix it's likely wrong ordering.

**Comment by aiChaoSONG** [comment]
@lgirdwood should be fixed by #2380

**Comment by aiChaoSONG** [comment]
close for fixed by #2380
